osi_lane.proto

```diff
@@ -346,6 +317,12 @@ message LaneBoundary
     //
     optional Color color = 4;
 
+    // The ids of \c StationaryObjects which limit the corresponding lane.
+    // This field must be set if the \c LaneBoundary.type is set to
+    // # TYPE_STRUCTURE
```

Comment:
Is there any reason for the '#' before TYPE_STRUCTURE?
![bildschirmfoto vom 2018-03-29 16-42-53](https://user-images.githubusercontent.com/26384662/38095253-4a549930-3370-11e8-8e81-331114a230e5.png)

Reply:
@carsten-kuebler can you check this?